app/build.gradle

```diff
@@ -139,6 +143,8 @@ afterEvaluate {
 }
 
 dependencies {
+    coreLibraryDesugaring 'com.android.tools:desugar_jdk_libs:1.0.9'
```

Comment:
Any harm in using the latest version instead?

Reply:
I'll update it, thanks.

Reply:
On that note, should I also update `debug.zip`?

Reply:
Leave it for the next release cycle, I think.